dali/kernels/test/warp_test/warp_test_helper.h

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) 2019, NVIDIA CORPORATION. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef DALI_KERNELS_TEST_WARP_TEST_WARP_TEST_HELPER_H_
+#define DALI_KERNELS_TEST_WARP_TEST_WARP_TEST_HELPER_H_
+
+#include "dali/kernels/imgproc/warp/affine.h"
+
+namespace dali {
+namespace kernels {
+
+/// @brief Apply correction of pixel centers and convert the mapping to
+///        OpenCV matrix type.
+inline cv::Matx<float, 2, 3> AffineToCV(const AffineMapping2D &mapping) {
```

Comment:
Should we collect all the OpenCV test helpers in one place?

Reply:
I don't know... this one is quite specific - i.e. it applies only to WarpAffine, but I don't think e.g. keypoint transformation should use the same fixup.